.github/workflows/pr.yaml

```
@@ -0,0 +1,15 @@
+      - main
+  workflow_dispatch:
+
+jobs:
+  build:
+    name: Build for PR
+    runs-on: ubuntu-latest
+    steps:
+      - name: Build app
+        uses: ./build-app.yaml
+        with:
+          artifact_name: 'pr-site'
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)